tutorials/distributed-keras-on-cmle/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Distributed Keras on Google Cloud Machine Learning Engine
+description: Learn how to use distributed Keras on Cloud Machine Learning Engine.
+author: corrieelston
+tags: Data Science, Cloud Machine Learning Engine, Keras, Python
+date_published: 2017-09-28
+---
+
+# Distributed Keras on Google Cloud Machine Learning Engine (CMLE)
+
+This solution connects the dots between a number of technologies with the end result of a production-grade model.
+
+First the technologies:
```

Comment:
You say "First[,] the technologies:", but there is no "next" or "second" following it. I suggest something more like, "This tutorial gives you an introduction to three machine learning technologies. Here is a brief description of each, along with links to more information about them:"